eng/test-configuration.json

```diff
@@ -5,22 +5,16 @@
   "retryOnRules": [
     {"testName": {"contains": "FlakyTestToEnsureRetryWorks" }},
     {"testName": {"contains": "MaxRequestBufferSizeTests.LargeUpload" }},
-    {"testName": {"contains": "ReaderThrowsResetExceptionOnInvalidBody" }},
-    {"testName": {"contains": "AttributeRouting_RouteNameTokenReplace_Reachable" }},
     {"testName": {"contains": "Http3RequestTests"}},
     {"testName": {"contains": "Http3TlsTests"}},
     {"testName": {"contains": "StreamPool_StreamAbortedOnClientAndServer_NotPooled"}},
-    {"testName": {"contains": "ServerCertificateSelector_Invoked"}},
     {"testName": {"contains": "TraceIdentifierIsUnique"}},
     {"testName": {"contains": "LocationChangingHandlers_CannotCancelTheNavigationAsynchronously_UntilReturning"}},
     {"testName": {"contains": "ClientAbortingConnectionImmediatelyIsNotLoggedHigherThanDebug"}},
-    {"testName": {"contains": "Listen_Http3AndSocketsCoexistOnSameEndpoint_AltSvcEnabled_Upgrade"}},
-    {"testName": {"contains": "Listen_Http3AndSocketsCoexistOnSameEndpoint_AltSvcDisabled_NoUpgrade"}},
     {"testName": {"contains": "WebConfigAppendsHostingStartup_OutOfProcess"}},
     {"testName": {"contains": "HttpsConnectionClosedWhenResponseDoesNotSatisfyMinimumDataRate"}},
     {"testName": {"contains": "BidirectionalStream_ServerReadsDataAndCompletes_GracefullyClosed"}},
     {"testName": {"contains": "POST_ServerAbort_ClientReceivesAbort"}},
-    {"testName": {"contains": "VariableMultipleStreamsInSequence"}},
```

Comment:
Just wondering if all of these tests are actually going to stop retrying❔ For example, are some covered by the blanket `Http3RequestTests`, `Http3TlsTests`, or `IIS` rules❔

Reply:
That's somewhat orthogonal, I'm just trying to reduce the number of lines on this list right now.  Keeping this file manageable is the main intent.

Reply:
Also retries are harmless if the test doesn't fail very often anyways, but this list growing huge is a maintenance headache no matter what